pkg/plugins/discovery/k8s/controllers/testdata/02.dataplane.yaml

```diff
@@ -14,7 +14,9 @@ spec:
     outbound:
       - address: 10.108.144.24
         port: 443
-        service: test-app.playground.svc:443
+        tags:
+          service: test-app.playground.svc:443
```

Comment:
I guess `test-app_playground_svc_443`?

Reply:
Yes, but I don't want to introduce it in this PR.